api/source/service/mysql/STIGService.js

```diff
@@ -969,7 +1049,34 @@ exports.getCci = async function(cci, userObject) {
  **/
 exports.getCcisByRevision = async function(benchmarkId, revisionStr, userObject) {
   try {
-    let rows = await _this.METHOD()
+
+    let [input, version, release] = /V(\d+)R(\d+(\.\d+)?)/.exec(revisionStr)
+
+    let sql = `
+      SELECT DISTINCT
+          c.cci,
+          c.type,
+        COALESCE((
+          SELECT JSON_ARRAYAGG(JSON_OBJECT(
```

Comment:
Same as above. Please include the other properties from the OAS definition:
```
        "version": "string",
        "location": "string",
        "indexDisa": "string",
        "textRefNist": "string",
        "parentControl": "string"
```
these are all in `crm` which is handy!

Reply:
added